### ISSUE TITLE
[BITV] 9.1.3.1e/3.1 - Although the table with directories and files visually has correct headings, these headings are being overridden by the use of the "aria-label" attribute, resulting in the content of data cells being announced with column headers like "Sort list by..." during screen reader output. The "aria-label" attribute should be removed, as the sorting status is conveyed through the "aria-sort" attribute. (2)

### ISSUE THREAD

**Issue by JuliaKirschenheuter** [nextcloud/server#41833]
![image](https://github.com/nextcloud/server/assets/6078378/9eafc8a2-ea72-47d4-8eae-0775a9369b31)

https://report.bitvtest.de/default-en/cbedb5ef-4815-4cba-a83d-995599c178c3.html#checkpoint-3f0ff0bc0e-v3-n1

9.1.3.1e/11.1 - https://report.bitvtest.de/default-en/cbedb5ef-4815-4cba-a83d-995599c178c3.html#checkpoint-3f0ff0bc0e-v11-n1
